content/en/examples/controllers/daemonset.yaml

```diff
@@ -35,6 +35,10 @@ spec:
         volumeMounts:
         - name: varlog
           mountPath: /var/log
+      # system-node-critical is the highest priority class and can be used to ensure that
+      # a critical daemonset's pod will evict an existing pod if that is required to allow
+      # the daemonset pod to schedule
+      priorityClassName: system-node-critical
```

Comment:
A bad example I'm afraid. Elasticsearch **isn't** node critical. I would omit this. It's OK to include a comment without actually setting the feild.

Reply:
I can remove the priority, but I thought this was just a fluentd DS exporting logs.  Not sure where the `fluentd_elasticsearch` org comes from.

Reply:
I just saw the edited comment, updated. Thanks!